src/include/cell_inline.h

```diff
@@ -1303,12 +1303,44 @@ __wt_cell_unpack_kv(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CELL
 }
 
 /*
- * __wt_cell_unpack_delta --
- *     Unpack a delta cell into a structure.
+ * __wt_cell_unpack_delta_int --
+ *     Unpack an internal delta cell into a structure.
  */
 static WT_INLINE void
-__wt_cell_unpack_delta(WT_SESSION_IMPL *session, const WT_DELTA_HEADER *dsk, WT_DELTA_CELL *cell,
-  WT_CELL_UNPACK_DELTA *unpack)
+__wt_cell_unpack_delta_int(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *page_dsk,
+  const WT_DELTA_HEADER *dsk, WT_DELTA_CELL *cell, WT_CELL_UNPACK_DELTA_INT *unpack_delta)
```

Comment:
There is still one problem. WT_DELTA_CELL is for the leaf page. We should separate this and have a WT_DELTA_CELL_INTERNAL. In the case of internal delta, we only have one flag, which is one byte.